### PR TITLE
Feature/pair status and layout

### PR DESCRIPTION
**Pull request by DamengRandom**
* `Pair Status` Panel UI
* `Flow` panel UI 
* Introduced `redux` for pair status tracking
* Pair screen page layout setup
* Pair page `responsive` display support
* Renamed interface start from `I`
* Extracted `DrawerList` and `NavbarHeader` styles logic into an independent file
* Removed unused counter redux related code

### CODE REVIEW

File: src/components/PairPage/index.styles.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      }),
+      marginRight: 0,
+    },
+    pairStatusContainer: {
+      marginBottom: -theme.spacing(2), // keep same position with configuration form save settings button
+    },
+    pairFormContainer: {
+      display: 'inherit',
+    },
+    '@media (max-width: 480px)': {
```
Comment (mattgresham) [inline src/components/PairPage/index.styles.ts:32]: To standardise these breakpoints, what do you think of using the theme helpers here - https://material-ui.com/customization/breakpoints/

File: src/components/Navbar/DrawerList/index.styles.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { TEXT_LIGHT_COLOR } from '../../../definitions/constants/themeStylesConfigs';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    list: {
+      width: 256,
+    },
+    title: {
+      color: TEXT_LIGHT_COLOR,
+      fontSize: 14,
```
Comment (mattgresham) [inline src/components/Navbar/DrawerList/index.styles.ts:12]: I noticed that the other font-size values are in rem, do we want to use rem everywhere?

**Review by mattgresham — APPROVED** [review]
Looks good!

I made a couple of minor things that you may want to think about

File: src/components/Navbar/DrawerList/index.styles.ts (hunk 2 above)
Comment (DamengRandom) [reply src/components/Navbar/DrawerList/index.styles.ts:12]: Oh yep, I will update it

File: src/components/PairPage/index.styles.ts (hunk 1 above)
Comment (DamengRandom) [reply src/components/PairPage/index.styles.ts:32]: Good point, I will try to make it happen ~

Comment (DamengRandom) [reply src/components/PairPage/index.styles.ts]: Thanks Matt, its a good suggestion and which works really well 👍

**Pull request merged**